archive.py

```diff
@@ -376,6 +378,9 @@ def write_last_updated(t):
     f.write('<hr><p>Last updated: {} UTC</p>'.format(t))
     f.close()
 
+def write_css():
+    copyfile('style.css', md_root / 'style.css')
```

Comment:
Let's use `os.path.join` for path manipulation.

Reply:
Using `/` has been done several times in archive.py, e.g. https://github.com/zulip/zulip_archive/blob/master/archive.py#L227. Should all of them be replaced with `os.path.join` too?

Reply:
I’m not sure what Tim has against `pathlib`. The only existing uses of `os.path.join` in this file are on `site_url`, which is actually wrong and seems likely to make Windows users unhappy; those should be `urllib.parse.urljoin`.

Reply:
Oh, I was in a hurry and hadn't noticed `md_root` was a pathlib thing and though this was just broken.  `pathlib` is fine.